.gitignore

```diff
@@ -14,7 +14,7 @@
 /.yardoc
 /_yardoc/
 /doc/
-
+/TAGS
```

Comment:
What's TAGS?

Reply:
Ugh, it’s one of those metadata files that are spat out by a vscode plugin I have. 

Perhaps there’s a way I can have it in a global ignore on my machine?

Reply:
Yeah you can https://stackoverflow.com/questions/45333484/how-to-use-global-gitignore-with-visual-studio-code

Reply:
Cool. You ok to yank this change from the PR? Or I can go find my laptop and do it